doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -3061,6 +3061,20 @@ Note that if you are attempting to validate a very large number of datasets in y
  
      asadmin set server-config.network-config.protocols.protocol.http-listener-1.http.request-timeout-seconds=3600
 
+
+
+Index and Database Consistency
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Get a list of all database objects that are missing in Solr, and Solr documents that are missing in the database::
+
+  curl $SERVER_URL/api/admin/index/status
+
+Remove all Solr documents that are orphaned (ie not associated with objects in the database)::
+
+  curl $SERVER_URL/api/admin/index/clear-orphans
+
+
```

Comment:
These docs look fine but I would prefer seeing them on this page: http://guides.dataverse.org/en/latest/admin/solr-search-index.html

It's nice having a single "clear-orphans" endpoint but is it fast enough? Will there be demand for a "clear dataset orphan"? I'm fine with putting "clear-orphans" out into the world and getting feedback about if more granularity is needed.

Reply:
moving the documentation to search-index.html.